kong/tracing/instrumentation.lua

```diff
@@ -229,7 +227,7 @@ function _M.request(ctx)
     http_flavor = string.format("%.1f", http_flavor)
   end
 
-  local active_span = tracer.start_span(span_name, {
+  local active_span = tracer.start_span("kong", {
```

Comment:
I notice that we are creating "root" spans for OTEL and Datadog plugins. It will not be created in most cases (unless we are not tracing, and receives no context headers).
Just to remind we need to take care of this at later refactoring.